src/api/test-controller/index.js

```diff
@@ -65,8 +66,9 @@ export default class TestController {
 
         this.testRun               = testRun;
         this.executionChain        = Promise.resolve();
-        this.callsitesWithoutAwait = new Set();
         this.warningLog            = testRun.warningLog;
+
+        globalCallsites.callsitesWithoutAwait = new Set();
```

Comment:
I don't think that we should reinitialize the set here. When running multiple browsers in parallel, there can be many TestController instances at the same time. We can lost some information here